.docker/ci/Dockerfile

```diff
@@ -1,50 +1,30 @@
-# moveit/moveit:melodic-ci
-# Sets up a base image to use for running Continuous Integration on Travis
+FROM ros:crystal-ros-base-bionic
```

Comment:
https://github.com/osrf/docker_images/blob/eb35e27e8a0842c8fb493bbdef37b314f931aa59/ros/crystal/ubuntu/bionic/desktop/Dockerfile#L5-L8

Reply:
Yes, we could use the desktop docker, but this would probably install a lot of unnecessary packages as well. I think installing packages by hand is fine, just use rosdep where possible.

Reply:
+1